client/signup/steps/store-features/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import intentImageUrl from 'calypso/assets/images/onboarding/intent.svg';
+import paymentBlocksImage from 'calypso/assets/images/onboarding/payment-blocks.svg';
+import wooImage from 'calypso/assets/images/onboarding/woo.png';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { saveSignupStep } from 'calypso/state/signup/progress/actions';
+import { shoppingBag, truck } from '../../icons';
+import SelectItems, { SelectItem } from '../../select-items';
+import './index.scss';
+
+interface Props {
+	goToNextStep: () => void;
+	isReskinned: boolean;
+	signupDependencies: any;
+	stepName: string;
+	initialContext: any;
+}
+
+export default function StoreFeaturesStep( props: Props ): React.ReactNode {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const headerText = translate( 'Setup your store' );
+	const subHeaderText = translate( "Let's create a website that suits your needs" );
+
+	const { stepName } = props;
+
+	// Only do following things when mounted
+	React.useEffect( () => {
+		dispatch( saveSignupStep( { stepName } ) );
+	}, [] );
+
+	const intents: SelectItem< any >[] = [
+		{
+			key: 'simple',
+			title: translate( 'Start Simple' ),
+			description: (
+				<>
+					<span className="store-features__requirements">{ translate( 'Free' ) }</span>
+					<span>
+						{ translate(
+							'Ideal if you’re looking to accept donations or sell one or two products without needing to manage shipping.'
+						) }
+					</span>
+					<span className="store-features__powered-by">
+						<span className="store-features__image-wrapper">
+							<img src={ paymentBlocksImage } alt="Payment Blocks" />
+						</span>
+						<span>
+							{ translate( 'Powered by' ) } <button>Payment Blocks</button>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'powered by' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feature/seller-experience-add-simple-vs-power-view)